client/lib/resource/src/create-scope/create-scope.js

```diff
@@ -10,22 +11,46 @@ class CreateScope extends BaseView {
         props.className = 'createScope';
         props.events = {
             'submit': 'save',
-            'keyup #scope_id': 'syncScopeId'
+            'keyup #scope_id': 'handleScopeId'
         };
         props.store = Flux.getStore('resource');
         super(props);
         this.actions = Flux.getActions('resource');
     }
 
- syncScopeId() {
+    handleScopeId() {
+        this.syncScopeId();
+        this.checkScopeIdAvailability();
+    }
+
+    syncScopeId() {
         this.$el.find('[data-action="sync-with-scope-id"]').text(this.$el.find('#scope_id').val());
     }
 
+    /**
+     * Checks the resource store if a scope with this ID
+     * already exists. Shows or hides according input-addon.
+     */
+    checkScopeIdAvailability() {
+        let {resourceId} = this.props;
+        let $scopeInput = this.$el.find('#scope_id');
+        let scope_id = $scopeInput.val();
+        if (Flux.getStore('resource').getScope(resourceId, scope_id)) {
```

Comment:
This assumes we already loaded all scopes for this resource. Which we probably have, just as a reminder.